modules/@apostrophecms/util/ui/public/2-http.js

```diff
@@ -360,4 +360,31 @@
       }
     }
   };
+
+  // apos.httpDraft is just apos.http decorated such that all calls
+  // are made with an apos-mode=draft query parameter
+  apos.httpDraft = {};
+  apos.util.assign(apos.httpDraft, apos.http);
```

Comment:
The point is for httpDraft to be a safe alternative to http, so I don't want developers scratching their heads over whether it has a particular method or not and just not using it. For that reason, and because it's just a shallow copy of a few properties, I think it makes sene this way.

Reply:
This doesn't feel great to me. Could we add an option (on `options`) to use make the request for a draft document? It'd get passed into `apos.http.remote` where all this decorating could be tidied into a helper function.